script/Ciel/Day 14/14_01B_03_CIEL14B_5.txt

```diff
@@ -508,40 +609,44 @@ That's why I know that he won't be able to handle the fact that I'm about to say
 [sha:d937fd562eddec1a0e97ed0df0d0204d4bb2b690]{
 -- Page 203, Offset 40850.
 -- 目を剥くような沈黙。
-A silence as if his eyes were wide open.
+There's a stunned silence.
 }
 [sha:20c3f33eca5cb772820315582bc7650549e530cc]{
 -- Page 203, Offset 40851.
 -- 振り返ってみれば。
-If I were to turn around and see him,
+Looking back on things―――
```

Comment:
Since JP doesn't give a shit about past/present/future shit this works, but it's probably better to be more explicit in EN
>Now that I think about it―――